docs/protocol/wallet-protocol.md

```diff
@@ -350,3 +370,190 @@ class RespondSESInfo(Streamable):
     reward_chain_hash: List[bytes32]
     heights: List[List[uint32]]
 ```
+
+:::note
+The below messages have been added via (Chip 26)[https://github.com/Chia-Network/chips/blob/8a597d06988eb308aa13488c5916ec041f39bc74/CHIPs/chip-0026.md].
```

Comment:
A separator before this might make it clearer it doesn't pertain to the above section. Also, the ()[] need to be reversed

Reply:
These are in a note so they have a separator associated with them. 
Thank you for catching the swap, that is now fixed